opentk/Makefile.include

```diff
@@ -25,59 +25,59 @@ $(addprefix $(MAC_BUILD_DIR)/mobile-%/,OpenTK.dll): XAMMAC = -r:$(MAC_BUILD_DIR)
 $(addprefix $(MAC_BUILD_DIR)/net_4_5/,OpenTK.dll): XAMMAC = -r:$(MAC_BUILD_DIR)/full-32/Xamarin.Mac.dll
 
 $(MAC_BUILD_DIR)/mobile-32/OpenTK.dll: $(MAC_OPENTK_SOURCES) $(MAC_BUILD_DIR)/mobile-32/Xamarin.Mac.dll
-	$(call Q_PROF_CSC,mac/$(VARIANT)) $(SYSTEM_MCS) \
+	$(call Q_PROF_CSC,mac/$(VARIANT)) $(MAC_mobile_CSC) \
 		-out:$@ -target:library -debug:full -define:OPENTK_DLL -unsafe -nowarn:3021,612,618,1635 \
 		$(MAC_BOOTSTRAP_DEFINES),COREBUILD \
 		$(XAMMAC) \
-		 -keyfile:$(PRODUCT_KEY_PATH) \
+		-publicsign -keyfile:$(PRODUCT_KEY_PATH) \
 		$(MAC_OPENTK_SOURCES)
 
 $(MAC_BUILD_DIR)/full-32/OpenTK.dll: $(MAC_OPENTK_SOURCES) $(MAC_BUILD_DIR)/full-32/Xamarin.Mac.dll
-	$(call Q_PROF_CSC,mac/$(VARIANT)) $(SYSTEM_MCS) \
+	$(call Q_PROF_CSC,mac/$(VARIANT)) $(MAC_full_CSC) \
 		-out:$@ -target:library -debug:full -define:OPENTK_DLL -unsafe -nowarn:3021,612,618,1635 \
 		$(MAC_BOOTSTRAP_DEFINES),COREBUILD \
-		-r:System.Drawing $(XAMMAC) \
-		 -keyfile:$(PRODUCT_KEY_PATH) \
+		-r:System.Drawing.dll $(XAMMAC) \
+		-publicsign -keyfile:$(PRODUCT_KEY_PATH) \
 		$(MAC_OPENTK_FULL_SOURCES)
 
 $(MAC_BUILD_DIR)/net_4_5/OpenTK.dll: $(MAC_OPENTK_SOURCES) $(MAC_BUILD_DIR)/full-32/Xamarin.Mac.dll
 	@mkdir -p $(MAC_BUILD_DIR)/net_4_5
-	$(call Q_PROF_CSC,mac/$(VARIANT)) $(SYSTEM_MCS) \
+	$(call Q_PROF_CSC,mac/$(VARIANT)) $(MAC_full_CSC) \
 		-out:$@ -target:library -debug:full -define:OPENTK_DLL -unsafe -nowarn:3021,612,618,1635 \
 		$(MAC_BOOTSTRAP_DEFINES),COREBUILD \
 		 $(XAMMAC) \
-		 -keyfile:$(PRODUCT_KEY_PATH) \
+		-publicsign -keyfile:$(PRODUCT_KEY_PATH) \
 		$(MAC_OPENTK_NET_4_5_SOURCES)
 
 $(MAC_BUILD_DIR)/net_4_5-reference/OpenTK.dll: $(MAC_BUILD_DIR)/net_4_5/OpenTK.dll
 	@mkdir -p $(@D)
 	$(Q) cp $^ $@
-	$(Q) cp $^.mdb $@.mdb
+	$(Q) cp $< $(@:.dll=.pdb)
```

Comment:
This doesn't look right, it will do:

```shell
cp $(MAC_BUILD_DIR)/net_4_5/OpenTK.dll $(MAC_BUILD_DIR)/net_4_5-reference/OpenTK.pdb
```

i.e. copy the dll to the pdb.

The same applies several times below.